packages/article-converter/src/plugins/embed/uuDisclaimerEmbedPlugin.tsx

```diff
@@ -15,7 +15,10 @@ import { type PluginType } from "../types";
 export const uuDisclaimerEmbedPlugin: PluginType = (element, opts, transformOpts) => {
   const props = attributesToProps(element.attribs);
   const data = JSON.parse(props["data-json"] as string) as UuDisclaimerMetaData;
-  const transformedDisclaimer = transform(data.embedData.disclaimer, transformOpts);
+  if (data.status === "error") {
+    return null;
```

Comment:
Burde vi ha bedre errorhåndtering her? vurderte å legge til støtte for uu-disclaimer i `EmbedErrorPlaceholder`, innspill?

Reply:
Synes det høres lurt ut, jeg. Evt bare vise innhold direkte?

Reply:
Hmm litt dumt å skjule uu-disclaimer helt kanskje? 🤔  eventuelt vise innholdet og legge en feilmelding over?